util/envDynawo.sh

```diff
@@ -1372,6 +1373,15 @@ nrt() {
   fi
 }
 
+nrt_clean() {
+  find $DYNAWO_NRT_DIR/data -depth -type d -name "outputs*" -not -path "*reference*/*" -exec rm -rf {} \; > /dev/null 2>&1
+  rm -rf $DYNAWO_NRT_DIR/output
+  rm -rf $DYNAWO_HOME/util/nrt_diff/output
+  find $DYNAWO_NRT_DIR -name "*.pyc" -exec rm -rf {} \;
+  find $DYNAWO_HOME/util/nrt_diff -name "*.pyc" -exec rm -rf {} \;
+  find $DYNAWO_NRT_DIR/data -empty -type d -delete
```

Comment:
I don't get this line? Is this to remove folders that still exist in local but were deleted in git?

Reply:
yes